### ISSUE TITLE
Creates new schemas files while typing !include definitions.

### ISSUE THREAD

**Issue by alvassin** [mulesoft/api-workbench#162]
Api workbench creates new schemas, while i type their paths. For example below, i can see two-three files created while i type full name:
E.g:
```
schemas:
  - myschema: !include schemas/myschema.json
```

Will create following files:
```
schemas/my
schemas/mysche
schemas/myschema
schemas/myschemas.json - this file is existing, i've created it before. all other files are redudant and i have to delete them.
```

Bug is not reproduced with examples, which i can also include in my document. Having issues while edition RAML 0.8 files.

**Comment by MelzerC** [comment]
Hey!

I have the exact same problem. While we are waiting for a fix I found a workaround to prevent the package from creating these files:
I installed the additional package `advanced-open-file`. Now, no more files are created and autocompletion works like a charm.

Crazy fun fact: After installing and disabling the additional package, no files are created.

**Comment by VasiliyLysokobylko** [comment]
Fixed in the 0.8.30
